vertica/tests/common.py

```diff
@@ -58,3 +56,14 @@
     'private_key': private_key,
     'ca_cert': CERTIFICATE_DIR,
 }
+
+
+def compose_file(vertica_version):
```

Comment:
nit: Not sure if this function is really "common" in the sense that's only used by conftest and not the tests themselves, but I'm not strongly opposed to it being here.

Reply:
Makes sense, `COMPOSE_FILE` was here and I didn't think of moving it.